galaxy.yml

```diff
@@ -10,7 +10,7 @@ namespace: worteks
 name: lemonldap
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 0.0.1
+version: 1.0.0-beta
```

Comment:
0.0.1 is a correct from a semantic versioning point of view, I'm not sure this is necessary.